example.json

```diff
@@ -0,0 +1,234 @@
+{
+    "type": "Feature",
+    "stac_version": "1.0.0",
+    "id": "resnet-18_sentinel-2_all_moco_classification",
+    "properties": {
+        "start_datetime": "1900-01-01",
+        "end_datetime": null,
+        "mlm:name": "Resnet-18 Sentinel-2 ALL MOCO",
+        "mlm:task": "classification",
+        "mlm:framework": "pytorch",
+        "mlm:framework_version": "2.1.2+cu121",
+        "mlm:file_size": 1,
+        "mlm:memory_size": 1,
+        "mlm:input": [
+            {
+                "name": "13 Band Sentinel-2 Batch",
+                "bands": [
+                    "B01",
+                    "B02",
+                    "B03",
+                    "B04",
+                    "B05",
+                    "B06",
+                    "B07",
+                    "B08",
+                    "B8A",
+                    "B09",
+                    "B10",
+                    "B11",
+                    "B12"
+                ],
+                "input_array": {
+                    "shape": [
+                        -1,
+                        13,
+                        64,
+                        64
+                    ],
+                    "dim_order": "bchw",
+                    "data_type": "float32"
+                },
+                "norm_by_channel": true,
+                "norm_type": "z_score",
+                "resize_type": "none",
+                "parameters": null,
+                "statistics": {
+                    "minimum": null,
+                    "maximum": null,
+                    "mean": [
+                        1354.40546513,
+                        1118.24399958,
+                        1042.92983953,
+                        947.62620298,
+                        1199.47283961,
+                        1999.79090914,
+                        2369.22292565,
+                        2296.82608323,
+                        732.08340178,
+                        12.11327804,
+                        1819.01027855,
+                        1118.92391149,
+                        2594.14080798
+                    ],
+                    "stddev": [
+                        245.71762908,
+                        333.00778264,
+                        395.09249139,
+                        593.75055589,
+                        566.4170017,
+                        861.18399006,
+                        1086.63139075,
+                        1117.98170791,
+                        404.91978886,
+                        4.77584468,
+                        1002.58768311,
+                        761.30323499,
+                        1231.58581042
+                    ],
+                    "count": null,
+                    "valid_percent": null
+                },
+                "norm_with_clip_values": null,
+                "pre_processing_function": "https://github.com/microsoft/torchgeo/blob/545abe8326efc2848feae69d0212a15faba3eb00/torchgeo/datamodules/eurosat.py"
```

Comment:
true. there's more to the pre-processing than just collate, I think I pointed to that file because they hardcode the mean and stds for normalization there. I saw this as more of a documentation field than something that would be used programmatically for processing. and since the API could change, thought pointing to a commit I used for preprocessing inputs for inference would be better. What do you think?